mirror_builder/__main__.py

```diff
@@ -15,12 +15,24 @@
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('toplevel', nargs='+')
     parser.add_argument('-v', '--verbose', action='store_true', default=False)
     parser.add_argument('-o', '--sdists-repo', default='sdists-repo')
     parser.add_argument('-w', '--wheels-repo', default='wheels-repo')
-    parser.add_argument('-t', '--work-dir', default=os.environ.get('WORK_DIR', 'work-dir'))
+    parser.add_argument('-t', '--work-dir', default=os.environ.get('WORKDIR', 'work-dir'))
     parser.add_argument('--wheel-server-port', default=0, type=int)
+    parser.add_argument('--isolate-builds', default=False, action='store_true')
```

Comment:
This is specifically network isolation right? As opposed to venv isolation? Maybe `--network-isolate`

And this is how we can add a temporary workaround for specific package builds that need network?